src/index.html

```diff
@@ -1,17 +1,195 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
+    <title>Layout HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="form__group">
+        <legend>Personal information</legend>
+        <div class="label__container">
+          <label for="surname">Surname:
+            <input
+              type="text"
+              id="surname"
+              name="surname"
+              autocomplete="off"
+              required
+              pattern="^[a-zA-Z]+$"
+            >
+          </label>
+        </div>
+        <div class="label__container">
+          <label for="name">Name:
+            <input
+              type="text"
```

Comment:
```suggestion
            >
          </label>
        </div>
        
        <div class="label__container">
          <label for="name">Name:
            <input
              type="text"
```

add blank line between multiline elements